mesh/mesh-io-tcpserver.c

```diff
@@ -218,6 +229,12 @@ static bool io_accept_callback(struct l_io *l_io, void *user_data)
 
 	server_fd = l_io_get_fd(l_io);
 
+#pragma GCC diagnostic push
```

Comment:
Please stick to the coding standard, even if we're not doing to submit this to upstream.